.github/workflows/post-merge-pipeline.yml

```diff
@@ -0,0 +1,156 @@
+# (C) 2023 GoodData Corporation
+
+name: Build web component's container image
+
+on:
+  push:
+    branches: ["master", "release", "hotfix/tiger", "hotfix/panther", "fast_track"]
```

Comment:
Should there be `rc` instead of release? Or are we planning to switch releases back to `release` branch?

Reply:
Matyas, how about with this draft plan for using branch `release` instead of `rc`:
1. use branch `release`, and we will have code drop every 2 weeks for web-components (for panther release)
2. For UI SDK release, doing git tag version on branch `release` (instead of branch `rc`)
3. Enable auto merge back from branch `release` to `master`
4. update tooling/job for UI SDK release according to branch `release`

Reply:
I'm not sure if we want to deploy web-components with each panther release?
To me, these are more related to SDK (and likely should be part of the SDK release process) rather than panther release.

Reply:
So it is still ok, just update the configuration for code drop tooling for gooddata-ui-sdk repos not every 2 weeks, but every month right? it is just complicating the process a bit, because most of components on Panther are updated every 2 weeks, just except web-components

Reply:
one more info, currently the Gitlab pipeline is also configured with the branch `release` so I did it here as the GitLab pipeline